painless_sqlalchemy/columns/AreaType.py

```diff
@@ -0,0 +1,63 @@
+import json
+from geoalchemy2 import WKTElement
+from painless_sqlalchemy.columns.AbstractType import AbstractType
+from painless_sqlalchemy.columns.AbstractGeometry import AbstractGeometry
+from painless_sqlalchemy.util.LocationUtil import validate_latlong
+
+
+class AreaType(AbstractGeometry, AbstractType):
+    """ Geo Polygon Column Type """
+
+    python_type = list
+    geometry_type = "POLYGON"
+
+    @classmethod
+    def enforce_clockwise(cls, area):  # reference: http://tiny.cc/eacajy
+        # check if clockwise
+        direction = 0
+        for i in range(len(area) - 1):
+            direction += (
+                area[i + 1][0] - area[i][0]
+            ) * (
+                area[i + 1][1] + area[i][1]
+            )
+        # reverse if counter clockwise
+        return area[::-1] if direction < 0 else area
```

Comment:
Should we raise an error here? Could be considered "magic"

Reply:
Thinking about naming here. We could have a flag `input_clockwise: bool` which would reverse the direction, but `enforce_clockwise` to me communicates "This is to make sure the input data is clockwise"

Reply:
Changed it to a boolean flag